### DISCUSSION TITLE
Build errors

### DISCUSSION THREAD

**Discussion by DeNelo** [mondeja/mkdocs-include-markdown-plugin#242] [General]
Hi,
I'm implementing your very, very useful plugin in our organization. Is there any way to make an error NOT stop the whole build process? For instance, if someone includes a file which isn't there, the build breaks. For some, it can be difficult to figure out why. Is there any way to not break the build, but rather add warnings to the log?

**Comment by mondeja** [comment]
Does this solution works for your usage case? https://github.com/mondeja/mkdocs-include-markdown-plugin/issues/228#issuecomment-2299959559 or #243

**Reply by DeNelo** [reply]
Thank you, I do believe it will!

**Comment by DeNelo** [comment]
Duplicate of #228
